lib/webview.dart

```diff
@@ -3,10 +3,10 @@ import 'package:webview_flutter/webview_flutter.dart';
 import 'package:url_launcher/url_launcher.dart';
```

Comment:
Getting a warning that launch from this package is deprecated and asks to use launchURL instead.

Reply:
Was depreciated in the newer version, we were using the old one, upgraded it